styles/style.css

```diff
@@ -92,11 +96,37 @@ p {
   border-left-color: var(--orange-dark);
 }
 
-
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+/* INSERT BUTTON STYLES HERE */
+
+button {
+  background-color: #e95b1e;
+  color: white;
+  border-radius: 4px;
+  padding: 6px 6px;
+  border: 0px;
+}
 
+button:hover {
+  background-color: rgb(55, 178, 235);
```

Comment:
You have used both RGB and hexadecimal color codes. To improve consistency, you can choose one format. Both RGB and hexadecimal color codes are valid and widely supported, so it's up to your personal preference.